src/history/hs.c

```diff
@@ -1538,7 +1562,7 @@ __hs_fixup_out_of_order_from_pos(WT_SESSION_IMPL *session, WT_CURSOR *hs_cursor,
          */
         if (__wt_txn_visible_all(
               session, hs_cbt->upd_value->tw.stop_txn, hs_cbt->upd_value->tw.durable_stop_ts))
-            continue;
+            goto while_end;
```

Comment:
This (and other instances of similar code style) might look ugly but I didn't want to have another call to `WT_HS_CUR_*()` macro for this special case. There is some good amount of code in these macros which will get copies around.

Reply:
Another idea would be to call the goto tag `prev` or `next` instead of things like `while_end` and `loop_end`. It signals your intent a bit better I'd say.

Reply:
I have changed the labels to more meaningful names.